borsh-derive/src/internals/attributes/item/mod.rs

```diff
@@ -199,4 +209,92 @@ mod tests {
         let actual = check_item_attributes(&item_enum);
         local_insta_assert_snapshot!(actual.unwrap_err().to_token_stream().to_string());
     }
+    #[test]
+    fn test_init_function() {
+        let item_struct = syn::parse2::<DeriveInput>(quote! {
+        #[derive(BorshSerialize, BorshDeserialize, PartialEq, Debug)]
+        #[borsh(init = initializonsdfasfsa)]
```

Comment:
What does that function name mean? I would prefer having descriptive names in tests to make the reviews easier

Reply:
`initializonsdfasfsa` supposed to mean any method that doesn't exist on struct/enum, but yeah it better to proper descriptive name